src/commands/release.ts

```diff
@@ -0,0 +1,38 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../oclif/CustomCommand'
+import appsRelease from '../modules/release'
+
+export default class Release extends CustomCommand {
+  static description =
+    'Bump app version, commit and push to remote. Only for git users. The first option can also be a specific valid semver version'
+
+  static examples = [
+    'vtex release',
+    'vtex release patch',
+    'vtex release patch beta',
+    'vtex release minor stable',
+    'vtex release pre',
+  ]
+
+  static flags = {
+    help: oclifFlags.help({ char: 'h' }),
+  }
+
+  static args = [
+    {
+      name: 'releaseType',
+      required: false,
+      default: 'patch',
```

Comment:
woooww, I think we can add here the options? That would be cool

Reply:
Good idea!